src/Components/Details/Details.css

```diff
@@ -13,22 +13,35 @@
 }
 
 .details_main {
-  height: 100vh;
+  max-height: 750px;
```

Comment:
Should maybe be a default styling method in life?

Reply:
It's certainly come in handy the last few days!